config/initializers/omniauth.rb

```diff
@@ -1,3 +1,5 @@
 Rails.application.config.middleware.use OmniAuth::Builder do
   provider :developer
-end
\ No newline at end of file
+  provider :instagram, 'INSTAGRAM_CLIENT_ID', 'INSTAGRAM_CLIENT_SECRET' 
```

Comment:
…did we need this? I thought this was handled in the `instagram.rb` file in the initializers folder.